app/scripts/directives/fromFile.js

```diff
@@ -178,14 +179,16 @@ angular.module("openshiftConsole")
             };
             return false;
           }
+          if (item.kind.endsWith("List")) {
```

Comment:
`$scope.isList` is set after the validation, but we can update the code so we set the variable before it.